### DISCUSSION TITLE
Dating on the blockchain - pros, cons, wildcards

### DISCUSSION THREAD

**Discussion by Oshyan** [PullDating/Org#3] [Revenue and Funding]
One possible model I mentioned in the post #2 for making sustainable infrastructure is using some kind of distributed approach that essentially mostly or entirely avoids the high costs of servers, bandwidth, etc. There are many possible incarnations of those, but one intriguing option worth at least exploring a bit is something like "Dating on the Blockchain", i.e. "Web 3 Dating". I'm sure others are working on this (that's a good research topic!), but again worth discussing here.

In addition to avoiding a majority of hosting costs, there is potential value around user identity and reputation management that is still friendly to some level of anonymity, marketing (Web 3 people would love it, instant market right there), potential collective ownership and contribution for supporting long-term development, etc.

Like I said, people are almost certainly already working on this, so we'd have to understand what is already in progress and whether we have any advantages over existing efforts. And whether those efforts will be solving the same problems we are. Crypto projects often seem to fall into the same traps of value (money) driven systems, even if they are in some way more egalitarian, etc. Ultimately the most important thing for us and the success of this project is that our incentives as an org are such that the user's goals essentially become our goals, on the larger, structural level. Or at the very least that our goals aren't in conflict with the user's goals.

**Comment by Oshyan** [comment]
Some advantages specifically in regards to DAO structures:
https://medium.com/coinmonks/web3-and-chill-38388c10f16a

https://truyou.io/
and their pitch (in Q&A) which lists a few potential advantages of this approach: https://www.reddit.com/r/algorand/comments/sajkh6/web3_dating_on_the_algorand_blockchain/

https://soldate.org/

https://apps.apple.com/ca/app/teleport/id1548799738

Old, outdated:
https://github.com/mojodating

**Comment by WilliamDormer** [comment]
Conceptually, I like the idea of a distributed computing system, but it is technically challenging, especially to do it privately and securely.
